app/views/layouts/application.html.erb

```diff
@@ -13,9 +13,9 @@
   </head>
 
   <body>
-    <% if !current_page?(controller: 'pages', action: 'home') %>
-      <%= render 'shared/navbar' %>
-    <% end %>
+
```

Comment:
Because, here you render the navbar for all pages EXCEPT home, then on the home page view, you render the navbar... That looks weird. Was there a particular reason for that ?

Reply:
Yes, because on the home page the navbar is inside the header, in order to have the picture on the background, and in this way it appears before the header